packages/xo-web/src/common/tooltip/index.js

```diff
@@ -11,6 +11,10 @@ import styles from './index.css'
 
 // ===================================================================
 
+// Tooltip wont be visible on disabled HTML elements, in this case wrap your element before wrapping it with this tooltip
```

Comment:
Move comment above `Tooltip` component.

//  Wrap disabled HTML elements before wrapping it with Tooltip ?

What do you think about adding  a simple example to be more clear and not take time to understand how to use it with disabled elements or do you think this comment is clear?

Reply:
I think that adding an example it's a good idea.
@pdonias what do you think ?

Reply:
Yes, something very simple, though:
```js
// <Tooltip>
//   <div>
//     <MyComponent />
//   </div>
// </Tooltip>
```

Also, you can say that it's not visible only in some browsers and if there's a link to the official information, that would be useful.

Reply:
Unfortunetly there's only this link that shows the problem https://devdocs.io/dom_events/mouseenter